### ISSUE TITLE
Artwork and metadata don't fetch consistently

### ISSUE THREAD

**Issue by epbesser** [bassrock/iTunesMetadata.bundle#1]
It works for some movies and doesn't for others. There is no immediate pattern for it. Is this plugin still being maintained? Thanks.

**Comment by bassrock** [comment]
Yes I actually use it as my main metadata. This plugin relies on the itunes purchase link being on trakt or findable.tv that support lookup by tvid and themoviedb id. Then from those sites it grabs the itunes store id and uses the iTunes Metadata api from there.

Do you have some examples on what it doesn't work on?

**Comment by epbesser** [comment]
Hi, thanks for your quick reply. I’d say it works on about 15 to 20 percent of my movies. A lot of the mainstream titles don’t work: godfather, avengers etc. Some others do work, including the first ones in the alien and avenger series, plus the whole Harry Potter series. TV shows don’t seem to work at the moment but I guess thats ok since plex doesn’t do square artworks I don’t think.

Thanks a lot for your help!

**Comment by raddacle** [comment]
I'm having a similar issue. One title that I just noticed didn't pickup itunes metadata is [Missing Link ](https://itunes.apple.com/us/movie/missing-link/id1458048658). Though you're right, the itunes link is not posted on trakt. Would it be possible to use [the API that Ben Dodson built](https://github.com/bendodson/itunes-artwork-finder) and is open source?
